frontends/p4/sideEffects.cpp

```diff
@@ -295,6 +336,8 @@ class DismantleExpression : public Transform {
         }
     };
 
+    // TODO: alias analysis would be generally useful... refactor this into
```

Comment:
Alias analysis would use def-use information, which needs the information computed by this pass.